src/test/end-to-end/shared/testingSupportApi.ts

```diff
@@ -18,22 +17,87 @@ const createFile = (filePath, fileName) => {
     };
 };
 
-export const createLocation = async (csvFile: string) => {
+export const createLocation = async (locationId: string, locationName: string) => {
     const token = await getDataManagementCredentials();
-
-    const filePath = path.join(os.tmpdir(), csvFile);
-    const file = createFile(filePath, csvFile);
     try {
         await superagent
-            .post(`${testConfig.DATA_MANAGEMENT_BASE_URL}/locations/upload`)
-            .set('enctype', 'multipart/form-data')
-            .set({ Authorization: 'Bearer ' + token.access_token })
-            .attach('locationList', file.file.body, file.file.name);
+            .post(`${testConfig.DATA_MANAGEMENT_BASE_URL}/testing-support/location/${locationId}`)
+            .send(locationName)
+            .set('Content-Type', 'application/json')
+            .set({ Authorization: 'Bearer ' + token.access_token });
     } catch (e) {
         throw new Error(`Failed to create location , http-status: ${e.response?.status}`);
     }
 };
 
+export const clearTestData = async () => {
+    await clearAllPublicationsByTestPrefix(testConfig.TEST_SUITE_PREFIX);
+    await clearAllSubscriptionsByTestPrefix(testConfig.TEST_SUITE_PREFIX);
+    await clearAllLocationsByTestPrefix(testConfig.TEST_SUITE_PREFIX);
+    await clearAllAccountsByTestPrefix(testConfig.TEST_SUITE_PREFIX);
+    await clearAllMediaApplicationsByTestPrefix(testConfig.TEST_SUITE_PREFIX);
+};
+
+export const clearAllPublicationsByTestPrefix = async (testSuitePrefix: string) => {
```

Comment:
The only test clean up method exposed outside here is clearTestData? If so the delete methods below does not need to have the 'export' prefix

Reply:
Removed export